components/dropdown/dropdown-content-mixin.js

```diff
@@ -739,36 +744,95 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 				minWidthOverride = mobileTrayMinWidthDefault;
 			}
 		}
+		if (mobileTrayBottom) {
+			minWidthOverride = 'calc(100vw - 2px)';
+		}
 
 		// set to max width
 		let widthOverride = this._width ? this._width : maxWidthOverride;
 
-		if (widthOverride && maxWidthOverride && widthOverride > (maxWidthOverride - 20)) widthOverride = maxWidthOverride - 20;
-		if (widthOverride && minWidthOverride && widthOverride < (minWidthOverride - 20)) widthOverride = minWidthOverride - 20;
+		if (!mobileTrayBottom) {
+			if (widthOverride && maxWidthOverride && widthOverride > (maxWidthOverride - 20)) widthOverride = maxWidthOverride - 20;
+			if (widthOverride && minWidthOverride && widthOverride < (minWidthOverride - 20)) widthOverride = minWidthOverride - 20;
+			maxWidthOverride = maxWidthOverride ? `${maxWidthOverride}px` : undefined;
+			minWidthOverride = minWidthOverride ? `${minWidthOverride}px` : undefined;
+		} else {
+			widthOverride = undefined;
+		}
+
+		let maxHeightOverride;
+		if (!specialMobileStyle) {
+			maxHeightOverride = this._contentHeight ? `${this._contentHeight}px` : 'none';
+		}
+		if (mobileTrayRightLeft) maxHeightOverride = '';
+		if (mobileTrayBottom) {
+			// default maximum height for tray (42px margin)
+			const mobileTrayMaxHeightDefault = window.innerHeight - 42;
```

Comment:
Similar to the left/right style, we want to always leave 42px of the backdrop visible at the top, and also adhere to the maxHeight if given by the consumer. The maxwidth is ignored, as it is set to 100%

Reply:
Should we move that `42` value up into a constant at the top of the file that they can all share? Might make changing it in the future easier.